setup.py

```diff
@@ -135,6 +135,7 @@
         "google-cloud-storage >= 1.32.0, < 3.0.0dev",
         "google-cloud-bigquery >= 1.15.0, < 3.0.0dev",
         "google-cloud-resource-manager >= 1.3.3, < 3.0.0dev",
+        "shapely < 2",
```

Comment:
Shouldn't this be a constraint rather than a requirement? I'm trying to figure out how to do it in a `setup.py` file.

Reply:
i don't know if setuptools.setup has a constraints option